tests/phpunit/tests/pluggable/wpMail.php

```diff
@@ -556,10 +556,9 @@ public function test_wp_mail_resets_properties() {
 	}
 
 	/**
-	 * Test that wp_mail() can send embedded images.
+	 * Tests that wp_mail() can send embedded images.
 	 *
 	 * @ticket 28059
-	 * @covers ::wp_mail
```

Comment:
`@cover` has already been added at the class level (see [https://github.com/mukeshpanchal27/wordpress-develop/blob/fix/28059-unit-tests/tests/phpunit/tests/pluggable/wpMail.php#L6](https://github.com/mukeshpanchal27/wordpress-develop/blob/fix/28059-unit-tests/tests/phpunit/tests/pluggable/wpMail.php#L6)), so there’s no need to add it to individual unit tests.